Sources/XCTest/XCTestCase.swift

```diff
@@ -48,6 +54,14 @@ private func test<T: XCTestCase>(testFunc: T -> () throws -> Void) -> XCTestCase
     }
 }
 
+// FIXME: These should be instance variables defined on XCTestCase, but when so
+//        defined Linux tests fail with "hidden symbol isn't defined". Use
+//        globals for the time being, as these seem to appease the Linux
+//        compiler.
+private var XCTLatestExpectationLocation: (file: StaticString, line: UInt) = ("", 0)
```

Comment:
Good call. On multiple occasions I came close to giving XCTestExpectation file and line attributes. I'll try consolidating on that approach and seeing how much cleaner this can get.